src/migrations/20231127184224-collect-close-reasons-objective.js

```diff
@@ -0,0 +1,38 @@
+const { CLOSE_SUSPEND_REASONS } = require('@ttahub/common');
+const { prepMigration, addValuesToEnumIfTheyDontExist } = require('../lib/migration');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      await queryInterface.renameColumn('Objectives', 'suspendReason', 'closeSuspendReason', { transaction });
+      await addValuesToEnumIfTheyDontExist(
+        queryInterface,
+        'enum_Objectives_suspendReason',
+        CLOSE_SUSPEND_REASONS,
+      );
```

Comment:
addValuesToEnumIfTheyDontExist has a bug that i did not see before, it is not using the transaction. We should fix this. It could be done in another PR so that all the migrations that use it can be updated also.

Reply:
I will fix this in the other PR, the user permissions one